vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -95,7 +86,7 @@
   </ItemDefinitionGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
-    <Import Project="$(SolutionDir)\packages\boost.1.68.0.0\build\boost.targets" Condition="Exists('$(SolutionDir)\packages\boost.1.68.0.0\build\boost.targets')" />
```

Comment:
>r)\pa [](start = 33, length = 5)

Is this change intentional?

Reply:
Yes. MSBuild ensures `SolutionDir` always has a trailing slash. Adding one is redundant.